datahub-web/@datahub/data-models/addon/constants/entity/person/tabs.ts

```diff
@@ -9,13 +9,64 @@ export enum PersonTab {
   UserOwnership = 'userownership',
   UserJitAccess = 'userjitaccess',
   UserUMPFlows = 'userumpflows',
-  UserFeatureList = 'userfeaturelist'
+  UserFeatureList = 'userfeaturelist',
+  UserLikeList = 'userlikelist',
+  UserSocialActionList = 'usersocialactionlist',
+  UserDYMIList = 'userdymilist'
 }
 
 /**
  * Tab properties available for Person entity
  */
-export const personTabProperties: Array<ITabProperties> = [TabProperties.userlists, TabProperties.userownership];
+export const personTabProperties: Array<ITabProperties> = [
```

Comment:
Doesnt this mean, we are exposing UMP flows as well as JIT? can you double check?

Reply:
We don't as the actual render props only take in UserOwnership. That being said, this is a minor leak and we'll have to figure out a better way to do tab properties in the future to address this.